facturacion_mexico/facturacion_fiscal/services/ffm_reconciliation.py

```
@@ -17,26 +17,40 @@
 	_extract_reconciliation_states,
 )
 from facturacion_mexico.facturacion_fiscal.api_client import get_facturapi_client
+from facturacion_mexico.facturacion_fiscal.cancellation_state import (
+	apply_cancellation_state,
+	derive_cancellation_reconciliation,
+	extract_canceled_at,
+)
 
 
-def _write_pac_response(sales_invoice_name, request_data, response_data, factura_fiscal_name):
+def _write_pac_response(
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `_write_pac_response`

(ANN202)